tcl/is.tcl

```diff
@@ -534,10 +266,58 @@ namespace eval qc::is {
         return [expr {[string toupper $bool] in {Y N YES NO TRUE FALSE T F 0 1}}]
     }
 
-    proc decimal {number} {
```

Comment:
Currently "is decimal 1.5e10" will return 1.

But "is decimal 1.5e10 11 1" will cause an error